src/main/webapp/config.js

```diff
@@ -5,7 +5,7 @@
  * @author tblanshard
```

Comment:
This file could be included in the .gitignore, so we don't disclose the project details.

Reply:
I have't realised we should this file in .gitigonore. Thanks for your suggestion :D.

Reply:
Modified .gitignore here https://github.com/googleinterns/step229-2020/pull/22/commits/3ba0a2ff46e7491182f0eb136bf26bd95bc3c4db